docs/introduction.md

```diff
@@ -23,7 +23,7 @@ A [STAC](https://github.com/radiantearth/stac-spec) and [OGC API Features](http:
  - `serve` to start the API server
 
 ### Database configuration
-By default  `franklin` attempts to connect to a database named `franklin` at `localhost:5432` with a password of `franklin`. Using the following CLI options you can customize your database connection:
+By default  `franklin` attempts to connect to a database named `franklin` at `localhost:5432` with a password of `franklin`. Using the following CLI options you can customize your database connection*:
```

Comment:
CI will get mad if these changes aren't build into the docs site. To do that:

- start an sbt shell with `./sbt`
- run `docs/mdoc` (this checks that the Scala examples in the markdown files with the current state of the Scala code)
- run `docs/docusaurusCreateSite` (this makes the updated markdown available in the `website/` directory where docusaurus expects it)

Reply:
This runs in the CI though, none of the built docs files are tracked in git, but I didn't see any changes in git after running the commands to build them locally (although they looked good locally).